packages/postgres-database/src/simcore_postgres_database/utils_folders.py

```diff
@@ -83,12 +85,19 @@ class InsufficientPermissionsError(FolderAccessError):
     msg_template = "could not find a parent for folder_id={folder_id} and gid={gid}, with permissions={permissions}"
 
 
+class UnexpectedFolderAccessError(FoldersError):
+    msg_template = (
+        "Unexpected None value for resolved_access_rights={resolved_access_rights} and last_exception={last_exception}."
```

Comment:
Is this going to the frontend? if yes how would a user do anything with this?
Nothing more descriptive, and maybe a tip for whoever reads it?

Reply:
agree